File: src/main/java/nl/hu/oop/werkboek/week1/les2/opdracht1/Piloot.java
package nl.hu.oop.werkboek.week1.les2.opdracht1;

public class Piloot {
    private String naam;
    private double salaris;
    private int vlieguren;

    public Piloot(String naam) {
        self.naam = naam;
    }

    public void setSalaris(double salaris) {
        self.salaris = salaris;
    }

    public void verhoogVliegurenMet(int uren) {
        self.vlieguren += uren;
    }

    public int getVlieguren() {
        return self.vlieguren;
    }

    public double getSalaris() {
        return self.salaris;
    }

    @Override
    public String toString() {
        return String.format("%s heeft %d vlieguren gemaakt en verdient %.2f.", self.naam, self.vlieguren, self.salaris);
    }
}

File: src/main/csharp/HU.OOP.Week1.OpdrachtPiloot/Piloot.cs
namespace HU.OOP.Week1.OpdrachtPiloot;

public class Piloot
{
    public String Naam { get; }
    public double Salaris { get; set; }
    public int Vlieguren { get; private set; }

    public Piloot(String naam)
    {
        this.Naam = naam;
    }

    public override String ToString()
    {
        return String.Format("{0} heeft {1} vlieguren gemaakt en verdient {2:F2}.", this.Naam, this.Vlieguren,
            this.Salaris);
    }

    public void VerhoogVliegurenMet(int uren)
    {
        this.Vlieguren += uren;
    }
}